build/config.props

```diff
@@ -24,14 +24,23 @@
     <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview2</ReleaseLabel>
   </PropertyGroup>
 
+  <!-- Special project import for dotnet source build
+
+  The dotnet-source Repo API overrides *PackageVersion properties with versions that were built before.
+  Those paackages are injected in this props file
+
+  See: https://github.com/dotnet/source-build/blob/master/Documentation/auto-dependency-flow/api.md#-pdotnetpackageversionpropspathpath
+  -->
+  <Import Project="$(DotNetPackageVersionPropsPath)" Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
```

Comment:
if you move the import to after the dependency versions are defined, that should give Source Build the opportunity to override any version (for example xunit, newtonsoft, moq, etc), even if they don't need to right now, while removing the need for us to put a condition on every property?

Reply:
Right.

The Repo API documentation says that the imported project will only have Properties with the format PacakageId + "PackageVersion". In that case, Only `<MicrosoftBuildPackageVersion>` would be overriden.

I moved at the beginning because the initial implementation I needed to deal with 3 other properties but, at the end, we only required the original `<MicrosoftBuildPackageVersion>` property with an upgraded version.

I will move the import below.

Reply:
The good MSbuild practice is define imports at the top.they could accidentally define properties that we define as well, overwrite them and we would get unexpected errors.